examples/SampleApp/Gemfile

```diff
@@ -1,13 +1,12 @@
 source 'https://rubygems.org'
 
-# You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
-ruby ">= 2.6.10"
-
```

Comment:
lets not remove this please.. this upper bound is to sway away from the mac default version

Reply:
@santhoshvai, sure, no problem. But can we at least bump it to v3? v2.6 is ridiculously old 😄

Reply:
sure, lets do that

Reply:
bumped to `3.1`, the one that we use on CI